lib/iris/experimental/animate.py

```diff
@@ -41,7 +41,9 @@ def animate(cube_iterator, plot_func, fig=None, **kwargs):
         object. See :meth:`iris.cube.Cube.slices`.
 
     * plot_func (:mod:`~iris.plot` or :mod:`~iris.quickplot` plot):
-        Plotting function used to animate.
+        Plotting function used to animate. Must accept the signature 
+        ``plot_func(cube, vmin, vmax, coords)``. contourf, contour, pcolor and
```

Comment:
It needs to be clear that the `vmin`, `vmax` and `coords` arguments are all supplied as keyword arguments (and thus can be in any order). It may also be wise to link to the functions in question as in

    :func:`~iris.plot.contourf`

I can't remember if this works well for `iris.quickplot` as well as `iris.plot`, perhaps that can be included too?